daemon/inspect_windows.go

```diff
@@ -11,7 +11,17 @@ func setPlatformSpecificContainerFields(container *Container, contJSONBase *type
 }
 
 func addMountPoints(container *Container) []types.MountPoint {
-	return nil
+	mountPoints := make([]types.MountPoint, 0, len(container.MountPoints))
```

Comment:
isn't this the same code that we have in `inspect_unix.go`? can you move it to `inspect.go`?